rocketpy/simulation/flight.py

```diff
@@ -2881,6 +2926,21 @@ def retrieve_temporary_values_arrays(self):
 
         return temporary_values
 
+    @cached_property
+    def get_controller_observed_variables(self):
+        """Retrieve the observed variables related to air brakes from the
+        controllers. If there is only one set of observed variables, it is
+        returned as a list. If there are multiple sets, the list containing
+        all sets is returned."""
+        observed_variables = [
+            controller.observed_variables for controller in self._controllers
+        ]
```

Comment:
I'm affraid that multiple controllers can generate duplicates in this list when added at to the same rocket. Would that be a problem?

Reply:
What do you mean by duplicates? This is iterating through all controllers, so it is meant to have multiple lists

Reply:
I find this method quite odd for these reason:

- Is it only useful as a workaround since the `Controller` is private? If yes I believe we should discuss a bit if it is useful creating another method that we will need to maintain for the sake of preventing interaction with the `Controller` class. 
    - My understanding (correct me if I am forgetting something important) is that if the name of the class is already private `_Controller`, then its attributes are "privated". Otherwise we would have to create methods in the `Flight` class for each of the `_Controller` attributes. 
- Properties are almost never named `get...`, they are the getters themselves;
- What is the reasoning behind this method being `cached`? It is not called frequently in the simulation nor it is a particularly slow method (generally there are few controllers).

Reply:
Ok, I understand the reasoning behind the method. Just be sure to ponder over what Gui said about the duplicates. I don't think here the duplicates would be a problem, but a simulation in which two different controllers change the same variable would be quite messy.

Last thing, I am not suggesting to simply let the user engage with the Controller, it is more that I would be ok with it. Your solution is more strict on not allowing breaking change, with is good.

Reply:
Thanks for solving this one.